XML/Other/ReplaceableTestParameters.xml

```diff
@@ -293,6 +293,11 @@ Scenario 2 : You need to run all the performance tests quickly.
 		<ReplaceThis>INFINIBAND_MVAPICH_NBC_TESTNAMES</ReplaceThis>
 		<ReplaceWith></ReplaceWith>
 	</Parameter>
+	<Parameter>
+		<!--  options include osu_latency, osu_bw etc. Refer http://mvapich.cse.ohio-state.edu/benchmarks/-->
+		<ReplaceThis>INFINIBAND_MVAPICH_OSU_P2P_TESTNAMES</ReplaceThis>
```

Comment:
is it possible to include multiple tests here?

Reply:
It wouldnt hurt to make these default: osu_latency, osu_bw, osu_bibw

Reply:
> is it possible to include multiple tests here?

Yes, we can add multiple tests like other lines. We can handle multiple values inside of test script.

Reply:
Added Multiple tests. Its is done a bit differently form IMB tests, as IMB takes test names as params whereas for OMB each test is a separate script.